css/properties/user-modify.json

```diff
@@ -0,0 +1,73 @@
+{
+  "css": {
+    "properties": {
+      "user-modify": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-moz-user-modify",
+          "support": {
+            "chrome": {
+              "prefix": "-webkit-",
+              "version_added": "1",
+              "notes": "Also supported: <code>-webkit-user-modify: read-write-plaintext-only</code> (Richtext will be lost)."
```

Comment:
@Elchi3 Is `read-user-modify` supposed to be a separate entry?

Reply:
Yes, `read-write-plaintext-only` should be a sub‑feature of `user-modify`, and listed as unsupported in Firefox.